src/pages/News.jsx

```diff
@@ -1,11 +1,134 @@
+import { Stack, Typography } from "@mui/material"
+import CampaignOutlinedIcon from "@mui/icons-material/CampaignOutlined"
+import Accordion from "@mui/material/Accordion"
+import AccordionSummary from "@mui/material/AccordionSummary"
+import AccordionDetails from "@mui/material/AccordionDetails"
+import IconButton from "@mui/material/IconButton"
+import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos"
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos"
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown"
+// import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import React, { useEffect } from "react"
 
+import Layout from "../components/Layout/Layout"
+import Title from "../components/shared/Title"
+import { NewsData } from "../data/NewsData"
 const News = () => {
+  // 연도와 월 데이터 추출하여 중복 제거
+  const years = [...new Set(NewsData.map(item => item.year))]
+  const months = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"]
+
+  const [selectedYear, setSelectedYear] = React.useState(years[0])
+  const [selectedMonth, setSelectedMonth] = React.useState(months[0])
+
+  const handlePreviousMonth = () => {
+    const monthIndex = months.indexOf(selectedMonth)
+    const previousMonth = monthIndex === 0 ? months[months.length - 1] : months[monthIndex - 1]
+    setSelectedMonth(previousMonth)
+    // 월 변경에 따라 연도 변경
+    if (previousMonth === months[months.length - 1]) {
+      // 이전 월이 1월인 경우 이전 연도로 변경
+      const currentYearIndex = years.indexOf(selectedYear)
+      const previousYear = currentYearIndex === 0 ? years[years.length - 1] : years[currentYearIndex - 1]
+      setSelectedYear(previousYear)
+    }
+  }
+
+  const handleNextMonth = () => {
+    const monthIndex = months.indexOf(selectedMonth)
+    const nextMonth = monthIndex === months.length - 1 ? months[0] : months[monthIndex + 1]
+    setSelectedMonth(nextMonth)
+    // 월 변경에 따라 연도 변경
+    if (nextMonth === months[0]) {
+      // 다음 월이 12월인 경우 다음 연도로 변경
+      const currentYearIndex = years.indexOf(selectedYear)
+      const nextYear = currentYearIndex === years.length - 1 ? years[0] : years[currentYearIndex + 1]
+      setSelectedYear(nextYear)
+    }
+  }
+
   return (
     <Layout>
-      <div>BirthDay</div>
+      <NewsWrapper>
+        <TitleWrapper>
+          <CampaignOutlinedIcon sx={{ fontSize: 40 }} />
+          <Title>MCY 소식</Title>
+        </TitleWrapper>
+        <SelectWrapper>
+          <IconButton onClick={handlePreviousMonth}>
+            <ArrowBackIosIcon />
+          </IconButton>
+          <Typography fontSize={15}>
+            {selectedYear}년 {selectedMonth}월
+          </Typography>
+          <IconButton onClick={handleNextMonth}>
+            <ArrowForwardIosIcon />
+          </IconButton>
+        </SelectWrapper>
+        <RenderingArea>
+          {NewsData.filter(item => item.year === +selectedYear && item.month === selectedMonth).map(item => (
```

Comment:
```suggestion
          {NewsData.filter(item => item.year === +selectedYear && item.month === selectedMonth).map(item => (
```
+통해서 숫자변환을 하기도하는데 더 눈에띄게 명시적인 방법은 `Number(selectedYear)` 이니까 참고하도록. 꼭 바꾸진 않아도됨.